consensus/config/src/crypto.rs

```diff
@@ -17,18 +17,16 @@ use shared_crypto::intent::INTENT_PREFIX_LENGTH;
 // to change all four aliases to point to concrete types that work with each other. Failure to do
 // so will result in a ton of compilation errors, and worse: it will not make sense!
 
-/// Network key signs network messages and blocks.
+/// Network key is used for TLS and network identity of the authority.
 pub type NetworkPublicKey = ed25519::Ed25519PublicKey;
 pub type NetworkPrivateKey = ed25519::Ed25519PrivateKey;
 pub type NetworkKeyPair = ed25519::Ed25519KeyPair;
-pub type NetworkKeySignature = ed25519::Ed25519Signature;
-pub type NetworkKeySignatureAsBytes = ed25519::Ed25519SignatureAsBytes;
 
-/// Protocol key is used in random beacon.
+/// Protocol key is used for signing blocks and verifying block signatures.
 pub type ProtocolPublicKey = bls12381::min_sig::BLS12381PublicKey;
-pub type ProtocolPublicKeyBytes = bls12381::min_sig::BLS12381PublicKeyAsBytes;
 pub type ProtocolPrivateKey = bls12381::min_sig::BLS12381PrivateKey;
 pub type ProtocolKeyPair = bls12381::min_sig::BLS12381KeyPair;
+pub type ProtocolKeySignature = bls12381::min_sig::BLS12381Signature;
```

Comment:
Just to confirm we may just use the ed25519 worker key for this instead?

Reply:
Yes we may switch to that in future.